content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
 Italy
 =====
 
-Allow the user to generate the EDI document for Italian invoicing.
+.. _italy/modules:
 
-This module allows the creation of the EDI documents and the communication with the SDICoop web
-service of FatturaPA to send or receive invoices and notifications.
+Configuration
+=============
+
+:doc:`Install <../../../../general/apps_modules>` the following modules to get all features of the
+Italian localization:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - Italy - Accounting
+     - `l10n_it`
+     - Default :doc:`fiscal localization package <../overview/fiscal_localization_packages>`
+   * - Italy - E-invoicing
+     - `l10n_it_edi`
+     - e-Invoice implementation
+   * - Italy - E-invoicing (SdiCoop)
+     - `l10n_it_edi_sdicoop`
+     - Web service e-Invoice implementation
+   * - Italy - Accounting Reports
+     - `l10n_it_reports`
+     - Country-specific reports
+   * - Italy - Transport Document (DDT)
+     - `l10n_it_stock_ddt`
+     - Transport Document (DDT)
 
 .. warning::
-   Once this module is installed, it's no longer possible to send invoices via :ref:`PEC mails
-   <italy/pec>`.
+   Once the `l10_it_edi_sdicoop` module installed, it is no longer possible to send invoices via
+   :ref:`PEC mails <italy/pec>`.
+
+.. image:: italy/italy-modules.png
+   :align: center
+   :alt: Italian localization modules
 
-SDICoop
-=======
+Company information
+-------------------
 
-Configuration
--------------
+Configuring the company's information ensures the proper set up of your Accounting database. To add
+information, go to :menuselection:`(Accounting --> Configuration -->) Settings --> General Settings`
+and in the :guilabel:`Companies` section, click :guilabel:`Update info`. From here, fill in the
+fields:
+
+- :guilabel:`Address`: the address of the company;
+- :guilabel:`VAT`: VAT of the company;
+- :guilabel:`Codice Fiscale`: the fiscal code of the company;
+- :guilabel:`Tax System`: the tax system under which the company falls;
+- :guilabel:`PEC address email`: the certified email address of the company.
+
+.. image:: italy/italy-company.png
```

Comment:
Ah, pardon, looking at this image, I notice not all the address fields are filled out.